### PR TITLE
Docs skin

### PR DESCRIPTION
**Pull request by alextrastero**
Work in progress, just so we can review later

### CODE REVIEW

File: docs/style.scss [hunk 1]
```diff
@@ -0,0 +1,890 @@
...
+      &.focused {
+        background-color: #eee;
+        outline: none;
+      }
+
+      font-size: 12px;
+      white-space: nowrap;
+    }
+  }
+}
```
Comment (alextrastero) [inline docs/style.scss]: this file should be the docs 'custom' css

File: docs/webpack.config.js [hunk 2]
```diff
@@ -1,34 +1,62 @@
...
+      { test: /\.css$|\.jpe?g$|\.gif$|\.png|\.ico$|\.woff2?$|\.ttf|\.eot|\.svg/, loader: 'file?name=[path][name].[ext]' },
+      { test: /\.scss$/, loader: ExtractTextPlugin.extract('style-loader', sassLoaders.join('!'))}
     ]
   },
   node: {
     fs: 'empty'
   }
 };
+
+module.exports = config
```
Comment (alextrastero) [inline docs/webpack.config.js]: i'm having issues adding the scss to the components

Comment (isaiah) [inline docs/webpack.config.js]: Do you mean the style.css file is not loaded on the doc site? I don't know how you can import `.scss` in `.js` file in webpack, do you have to load the `style.css` somewhere in `Root.js`?

Comment (alextrastero) [inline docs/webpack.config.js]: with the loader, it should be as easy as import 'comp/button.scss'. Gonna have a look into it today

**Pull request closed without merging**